integration/test-mb/test-amqp/src/test/java/org/wso2/ei/mb/test/client/QueueMessageListener.java

```diff
@@ -43,6 +44,11 @@
     private int delay = 0;
 
     /**
+     * Acknoledgement mode
+     */
+    private int acknowledgeMode = 0;
```

Comment:
Could you describe what each acknowledgment mode value represents ? or usage of enum would be more appropriate for this ?